Automattic/sensei
# Update Sensei LMS icon in WordPress admin sidebar

Fixes #4306.

### Changes proposed in this Pull Request
- Uses a data URI to update the Sensei LMS icon in the WordPress admin sidebar.
- Updates the icons in the _At a Glance_ dashboard panel. (Previously, the old course icon used the graduation hat, which is the old Sensei icon.)

I had to manipulate the original SVG to get the colors to match the ones in the sidebar, so although the `assets/css/images/sensei-admin-menu.svg` is not included in the build, I'm committing it in case we need to manipulate it again in the future.

### Testing instructions
- Ensure the Sensei LMS icon has been updated in the WordPress admin sidebar.
- Go to _Dashboard_ > _Home_. In the _At a Glance_ dashboard panel, ensure the course and lesson icons match the ones in the sidebar.
- Test on multiple browsers.

### Screenshot / Video
![Screen Shot 2021-09-14 at 1 09 03 PM](https://user-images.githubusercontent.com/1190420/133302728-9a7c3c49-5f3a-4399-81bd-441fee1ec64e.jpg)
